spec/models/customer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+RSpec.describe Customer, type: :model do
+  subject { Customer.new(first_name: "Jack", last_name: "Smith", phone: "8889995678", email: "jsmith@sample.com" )}
+  it "is valid with valid attributes" do
+    expect(subject).to be_valid
+  end
+  it "is not valid without a first_name" do
+    subject.first_name=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a last_name" do
+    subject.last_name=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a phone number" do
+    subject.phone=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without an email" do
+    subject.email=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid if the phone number is not 10 chars" do
+    subject.phone="123456789"
+    expect(subject).to_not  be_valid
+  end
+  it "is not valid if the phone number is not all digits" do 
+    subject.phone="abcdefghij"
```

Comment:
I would add another case here that includes some digits but also some text, like` 1-800-call-me` or something